update_pipeline_details.php

```diff
@@ -42,14 +43,37 @@ function get_gh_api($gh_api_url) {
     ]);
 
     // Get API response
-    $gh_api_raw = file_get_contents($gh_api_url, false, $gh_api_opts);
-    if (strpos($http_response_header[0], 'HTTP/1.1 200') === false) {
-        die("\n-------- START ERROR " . date('Y-m-d h:i:s') . " --------\nCould not fetch $gh_api_url \n");
-        var_dump($http_response_header);
-        echo "\n$gh_api_raw\n";
-        die("\n-------- END ERROR " . date('Y-m-d h:i:s') . " --------\nCould not fetch $gh_api_url \n");
+    $first_page = true;
+    $next_page = false;
+    $res = [];
+    while ($first_page || $next_page) {
+        // reset loop vars
+        $first_page = false;
+        // Get GitHub API results
+        if ($next_page) {
+            $gh_api_url = $next_page;
```

Comment:
Where is this variable coming from? Just want to check that we're not breaking stuff by overwriting it here..

Reply:
it's the input parameter to this github query function